joomla/joomla-cms
# [5.1] Convert mod_frontend to service provider

### Summary of Changes

Converts the frontend link administrator module to service provider.


### Testing Instructions

1. Make sure you have administrator module mod_frontend published
2. You should see module output

![mod_messages](https://github.com/joomla/joomla-cms/assets/59166858/e2d06be6-ae11-4fff-9514-6d2b2a940585)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
